SSM/GameManagers/GameManager.java

```diff
@@ -528,18 +528,16 @@ public void PlayerQuit(PlayerQuitEvent e) {
     public void clickEvent(InventoryClickEvent e) {
         Player player = (Player) e.getWhoClicked();
         ItemStack item = e.getCurrentItem();
-        if (item == null) {
-            return;
-        }
-        if (!e.getView().getTitle().contains("Vote")) {
-            return;
-        }
+        if (item == null) return;
+        if (!e.getView().getTitle().contains("Choose")) return;
```

Comment:
Technically we should be using static constants for these probably to make sure they don’t change behavior, but to be smart this should probably be checking “Map” not “Choose” since it could overlap with other menus.